daml-lf/integration-test-lib/src/main/com/daml/lf/CantonRunner.scala

```diff
@@ -49,10 +53,7 @@ object CantonRunner {
             Vector.fill(nParticipants)(LockedFreePort.find() -> LockedFreePort.find())
           val domainPublicApi = LockedFreePort.find()
           val domainAdminApi = LockedFreePort.find()
-
-          val cantonPath = rlocation(
-            "external/canton/lib/canton-open-source-2.7.0-SNAPSHOT.jar"
-          )
+          val jarPath = if (devMode) cantonDevPath else cantonPath
```

Comment:
I don't like that `devMode` gets into scope from  `import config._`
But we can make a future PR to improve the clarity of this.